lib/transformations/index.js

```diff
@@ -1,7 +1,11 @@
 const loaderTransform = require('./loaders/loaders');
 const resolveTransform = require('./resolve/resolve');
+const removeJsonLoaderTransform = require('./removeJsonLoader/removeJsonLoader');
+const uglifyJsPluginTransform = require('./uglifyJsPluginTransform/uglifyJsPluginTransform');
```

Comment:
What do you mean? I'd add `Transform` suffix to all transforms just to make it more explicit. Thoughts?